docs/dev/tools/common-errors.md

```diff
@@ -184,27 +184,13 @@ This can happen when there is a slow internet connection, when the module is try
 **Solution:**
 
 - Try using a faster internet connection.
-- If the problem persists, try increasing the `VIAM_MODULE_STARTUP_TIMEOUT` or `VIAM_RESOURCE_CONFIGURATION_TIMEOUT` environment variables in the module's JSON configuration file.
-  For example:
-
-  ```json {class="line-numbers linkable-line-numbers"}
-  {
-    "modules": [
-      {
-        ...
-        "env": {
-          "VIAM_MODULE_STARTUP_TIMEOUT": "6m30s",
-          "VIAM_RESOURCE_CONFIGURATION_TIMEOUT": "3m0s"
-        }
-      }
-    ]
-  }
-  ```
-
 - If you are the module author, consider packaging the module with required dependencies so they don't need to be downloaded on startup.
   For Python modules, you can package your module with dependencies by using the PyInstaller steps when [uploading your module](/operate/get-started/other-hardware/#upload-your-module).
+- If the problem persists, try setting the `VIAM_MODULE_STARTUP_TIMEOUT` or `VIAM_RESOURCE_CONFIGURATION_TIMEOUT` environment variables on your machine.
+  For a value, pass a sequence of numbers and units, e.g. "6m30s", "6.5m", or "630000ms".
```

Comment:
```suggestion
  For a value, pass a sequence of numbers and units, for example `6m30s`, `6.5m`, or `630000ms`.
```
(Or optionally, include quotes inside the backticks if the thing must be passed with quotes as a string)

Reply:
```suggestion
  For a value, pass a sequence of numbers and units, for example "6m30s50ms" or "5m".
```